packages/grafana-schema/src/schema/graph.gen.ts

```diff
@@ -78,6 +81,10 @@ export interface LineStyle {
   fill?: 'solid' | 'dash' | 'dot' | 'square';
 }
 
+export const defaultLineStyle: Partial<LineStyle> = {
+  dash: [],
+};
+
```

Comment:
Default values for lists are a rough edge in cuetsy atm. And in my mind, too. While it's obvious to me that a present field containing an empty string is not the same as an absent field, it's not similarly clear with an empty list. (Or an empty struct, for that matter). This is an underspecified, rough edge in CUE as well, something i've discussed a bit with the authors.

Because it's both unclear to me, and there's crisp answer from CUE itself yet (the [output](https://gist.github.com/sdboyer/36a526ec011b5568f4ca6691a6b58790) of [this test](https://github.com/grafana/cuetsy/blob/c119b17908bf0b1d863b1b439d7da755b6913bad/analyze_test.go#L364) let me at least grok CUE's range of behavior), i've let cuetsy pretty much always generate an empty list as a default, even when a default isn't specified on input.

If we think this is a problem for Grafana, i should be able to add some controls over behavior.